lib/mappers/figma/border.js

```diff
@@ -0,0 +1,41 @@
+const {mapColorKeys} = require('../../utils');
+const formatColor = require('../sketch/color');
+const {tokenize} = require ('../../utils');
+const mapGradient = require('./gradient');
+
+const getGradientPropertyLocation = (border) =>  border.children[0].strokes[0];
+exports.getGradientPropertyLocation = getGradientPropertyLocation;
+
+const getBorderInfo = (border) => {
+    const info = border.children[0];
+    return {
+        width: info.strokeWeight,
+        type: info.strokeAlign,
+        radius: border.name.toLowerCase().includes('gradient') 
+        ? info.rectangleCornerRadii || [0]
```

Comment:
hmm having `number[]` here implies that it should also be available like this in Sketch in this format. Output for Sketch & Figma should be as 1:1 as possible